.gitignore

```diff
@@ -1 +1,4 @@
-/data
\ No newline at end of file
+/data
+.vscode
+app
```

Comment:
where is `app` coming from though?

Reply:
That's the built binary, should be ignored in. my opinion